x/stakeibc/keeper/icqcallbacks_callibrate_delegation.go

```diff
@@ -54,36 +57,6 @@ func CalibrateDelegationCallback(k Keeper, ctx sdk.Context, args []byte, query i
 		return errorsmod.Wrapf(types.ErrValidatorNotFound, "no registered validator for address (%s)", queriedDelegation.ValidatorAddress)
 	}
 
-	// Check if the ICQ overlapped a delegation, undelegation, or detokenization ICA
-	// that would have modfied the number of delegated tokens
-	prevInternalDelegation := callbackData.InitialValidatorDelegation
-	currInternalDelegation := validator.Delegation
-	icaOverlappedIcq, err := k.CheckDelegationChangedDuringQuery(ctx, validator, prevInternalDelegation, currInternalDelegation)
-	if err != nil {
-		return err
-	}
-
-	// If the ICA/ICQ overlapped, submit a new query
-	if icaOverlappedIcq {
-		// Store the updated validator delegation amount
-		callbackDataBz, err := proto.Marshal(&types.DelegatorSharesQueryCallback{
-			InitialValidatorDelegation: currInternalDelegation,
-		})
-		if err != nil {
-			return errorsmod.Wrapf(err, "unable to marshal delegator shares callback data")
-		}
-		query.CallbackData = callbackDataBz
-
-		if err := k.InterchainQueryKeeper.RetryICQRequest(ctx, query); err != nil {
-			return errorsmod.Wrapf(err, "unable to resubmit delegator shares query")
-		}
-		return nil
-	}
-
-	// If there was no ICA/ICQ overlap, update the validator to indicate that the query
-	//  is no longer in progress (which will unblock LSM liquid stakes to that validator)
-	validator.SlashQueryInProgress = false
```

Comment:
I think I agree with removing this flag completely from the calibration query (i.e. not setting it before the submission and ignoring it in the callback). 

But one downside is that we we still need to set back the queries that are currently stuck. We can do this in the upgrade handler, but there's the potential that another gets stuck before the upgrade occurs

Reply:
I don't recall the convo about removing this from the calibration query. Could you point me to that?

Reply:
I don't recall having a convo about it, I just saw it in this PR haha

Reply:
sorry for confusion - I'm suggesting I agree with the change.

---

I think there's two ways to do this:
1. Keep setting flag during calibration query
    * Pros: we can run a query to set all the current "true" flags back to false
    * Cons: opens the door for them to get stuck again
2. Do not set flags at all during calibration query (implementation in this PR)
    * Pros: we don't risk the flag getting stuck to true
    * Cons: we need to manually set current stuck flags back to false

---

But I'm saying I agree with option 2 - I don't think we actually have real pending queries, so we should be able to just set all flags back to false in the upgrade handler

Reply:
Sounds good! Seems reasonable to flip those flags in the upgrade handler

Reply:
Tracking it [here](https://github.com/Stride-Labs/stride/issues/1003) - marking this resolved!